platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -1644,6 +1668,45 @@ MGL_EXPORT
  */
 @property (nonatomic, null_resettable) NSExpression *textVariableAnchor;
 
+/**
+ The property allows to control an orientation of a symbol. Note, that the
+ property values act as a hint, so that Symbols whose language doesn't support
```

Comment:
Nit:

> Note that the property values act as a hint, so that a symbol whose language doesn’t support the provided orientation will be laid out in its natural orientation.